tensorflow/python/ops/image_ops_impl.py

```diff
@@ -792,6 +792,15 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     ValueError: If the shape of `image` is incompatible with the `offset_*` or
       `target_*` arguments, or either `offset_height` or `offset_width` is
       negative.
+      
+  Usage example:
+  ```python
```

Comment:
No need to add the ` ```python` and ` ``` ` markers. Just have an empty line after `Usage example:` and after the code block. Move the doctest above, just before the `Args:` section